src/Integration API/Controllers/DrugstoreController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Integration.Repository.Sql;
+using Integration_API.Model;
+//using Integration.Model;
+using Model.DataBaseContext;
+
+namespace Integration_API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class DrugstoreController : ControllerBase
+    {
+        private readonly MyDbContext dbContext;
+        public DrugstoreSqlRepository repo = new DrugstoreSqlRepository();
+        
+        public DrugstoreController(MyDbContext db) //Ovo mora da stoji, ne znam zasto!!!
+        {
+            this.dbContext = db;
+        }
+
+        [HttpGet]       // GET /api/drugstore
+        public IActionResult Get()
+        {
+            repo.dbContext = dbContext;
+            List<Integration_API.Model.Drugstore> result = new List<Integration_API.Model.Drugstore>();
+            repo.GetAll().ForEach(drugstore => result.Add(new Drugstore(drugstore.Id, drugstore.Name, drugstore.Url, drugstore.ApiKey)));
```

Comment:
ApiKey missing in table.

Reply:
Ovo ti je bacalo jer ApiKey nije bio u bazi, kad budes pokretao moras da podesis validnu bazu podataka inace ti nece raditi.